lib/engine/game/g_rolling_stock/game.rb

```diff
@@ -1043,6 +1043,26 @@ def movement_chart(corporation)
           chart << ['', ''] while chart.size < 5
           chart
         end
+
+        def liquidity(player, emergency: false)
+          total = player.cash
+          return(total) unless @round&.stock?
```

Comment:
What's your thoughts on always returning the stock round liquidity? I find it helpful to know when determining dividends and what to start a company at.

Reply:
Yeah, probably makes sense.